src/core/components/RootNavLayout/RootNavLayout.tsx

```diff
@@ -0,0 +1,15 @@
+import { Outlet } from 'react-router-dom';
+import { NavMenu } from '../NavMenu';
+import './RootNavLayout.css';
+
+export const RootNavLayout = (): JSX.Element => {
+  return (
+    <div className="root-layout-container">
+      <div className="root-layout-header"></div>
+      <div className="root-content-container">
+        <Outlet />
+      </div>
+      <NavMenu />
```

Comment:
Please add a container for the footer area. This container should define area dimensions and location. The footer area might contain more than 1 component

Reply:
Footer container added.